pkgs/by-name/ti/tilt/binary.nix

```diff
@@ -21,16 +23,30 @@ buildGoModule rec {
 
   ldflags = [ "-X main.version=${version}" ];
 
+  nativeBuildInputs = [ installShellFiles ];
+
   preBuild = ''
     mkdir -p pkg/assets/build
     cp -r ${tilt-assets}/* pkg/assets/build/
   '';
 
+  postInstall = '''';
+
   meta = {
     description = "Local development tool to manage your developer instance when your team deploys to Kubernetes in production";
     mainProgram = "tilt";
     homepage = "https://tilt.dev/";
     license = lib.licenses.asl20;
     maintainers = with lib.maintainers; [ anton-dessiatov ];
   };
-}
+}).overrideAttrs
+  (old: {
+    postInstall =
```

Comment:
It didn't seem like `postInstall` is a supported argument of `buildGoModule`. sorry if the way I've done this is bad form in nixpkgs

Reply:
Also, lmk if I should do [the fancy stdenv.buildPlatform.canExecute thing](https://github.com/NixOS/nixpkgs/blob/bedcba84aa20c95c7f0afc24606574ee5d13ca6a/pkgs/by-name/ar/argocd/package.nix?plain=1#L60)

Reply:
It is needed for cross-compiling.

Reply:
`postInstall` is a valid argument for buildGoModule, it will be forwarded to mkDerivation.